_posts/2022-09-21-hvr-and-hva.markdown

```diff
@@ -0,0 +1,110 @@
+---
+layout: post
+font: serif
+title: HVR and HVA
+date: 2022-08-08
+highlight: monokai
+author: Matt Carter, Jack Menzie, Leon Ticharwa
+image: /img/blog/hvr-and-hva/banner.jpg
+tags: hvr hva data replication
+---
+
+# HVR and HVA
```

Comment:
Good reference for HVA
https://www.fivetran.com/blog/high-volume-replication-is-here-starting-with-oracle